### PR TITLE
[13.0][MIG] report_wkhtmltopdf_param

### PR DESCRIPTION
**Pull request by Saran440**
Standard Migrate

### CODE REVIEW

**Comment by etobella** [comment]
@Saran440 Add `  - export WKHTMLTOPDF_VERSION=0.12.4` on `.travis.yml` like in https://github.com/OCA/account-financial-reporting/blob/12.0/.travis.yml

Wkthmltopdf is not installed by default and this is giving issues

**Review by etobella — COMMENTED** [review]
LGTM :+1: No major changes

**Review by etobella — APPROVED** [review]

**Comment by Saran440** [comment]
@etobella Fixed, Thank you for your advise :)

File: report_wkhtmltopdf_param/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+{
+    "name": "Report Wkhtmltopdf Param",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "summary": """
+        Add new parameters for a paper format to be used by wkhtmltopdf
+        command as arguments.
+    """,
+    "author": "Avoin.Systems, Eficent, Odoo Community Association (OCA)",
+    "website": "https://avoin.systems",
```
Comment (newtratip) [inline report_wkhtmltopdf_param/__manifest__.py]:
```suggestion
    "website": "https://github.com/OCA/reporting-engine",
```

File: report_wkhtmltopdf_param/models/report_paperformat.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+
+class Paper(models.Model):
+    _inherit = "report.paperformat"
+
+    custom_params = fields.One2many(
+        "report.paperformat.parameter",
+        "paperformat_id",
+        "Custom Parameters",
+        help="Custom Parameters passed forward as wkhtmltopdf " "command arguments",
```
Comment (newtratip) [inline report_wkhtmltopdf_param/models/report_paperformat.py]:
```suggestion
        help="Custom Parameters passed forward as wkhtmltopdf command arguments",
```

**Review by newtratip — COMMENTED** [review]
Some comments.

**Comment by Saran440** [comment]
@newtratip Fixed, Thank you for your reviewed.

**Review by newtratip — APPROVED** [review]
LGTM :+1:

**Pull request merged**